src/NativeShell/www/apphost.js

```diff
@@ -0,0 +1,172 @@
+//disables all console logs
+console.log = function () {};
+
+var deviceId;
+var deviceName;
+var appName;
+var appVersion;
+
+var features = [
+    //'filedownload',
+    'sync',
+    'customsyncpath',
+    'displaylanguage',
+    'subtitleappearancesettings',
+    'cameraupload',
+    'sharing',
+    'exit',
+    'htmlaudioautoplay',
+    'htmlvideoautoplay',
+    'externallinks',
+    'multiserver',
+    'physicalvolumecontrol',
+    'remotecontrol',
+    'castmenuhashchange'
+];
+
+function getDeviceProfile(profileBuilder) {
+    var profile = profileBuilder();
+
+    profile.DirectPlayProfiles.push({
+        // TODO investigate ac3 support
+        Container: "m4v,3gp,ts,mpegts,mov,xvid,vob,mkv,wmv,asf,ogm,ogv,m2v,avi,mpg,mpeg,mp4,webm,wtv",
+        Type: 'Video',
+        AudioCodec: 'aac,aac_latm,mp2,mp3,wma,dca,dts,pcm,PCM_S16LE,PCM_S24LE,opus,flac'
+    });
+
+    profile.CodecProfiles = profile.CodecProfiles.filter(function (i) {
+        return i.Type == 'Audio';
```

Comment:
huh? what this does? is this function poorly named?

Reply:
Well it's also named this way in jf-web. Should provide a set of supported formats, not sure exactly what's for. I just kept original functionality not to break anything

Reply:
Just to be sure, are you talking about getDeviceProfile?

Reply:
I'm wondering why `profile.CodecProfiles` contains only audio codecs, not video or smth else...

Reply:
For what i see, the video codecs are added after that filtering. I can't give you a direct answer to why this was done this way.